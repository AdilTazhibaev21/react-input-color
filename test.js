import React from 'react';
import renderer from 'react-test-renderer';
import InputColor from './';

test('render', () => {
  const component = renderer.create(<InputColor value="#3498db" />);
  expect(component.toJSON()).toMatchSnapshot();
});
