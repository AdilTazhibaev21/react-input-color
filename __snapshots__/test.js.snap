// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render 1`] = `
<span
  className="m-input-color"
>
  <span
    className="css-color"
    onClick={[Function]}
    style={
      Object {
        "background": "rgba(52,152,219,1)",
      }
    }
  />
  <span
    className="remove"
    onClick={[Function]}
  >
    Ã—
  </span>
</span>
`;
